applications/StructuralMechanicsApplication/custom_constitutive/linear_isotropic_damage_plane_strain_2d.cpp

```diff
@@ -0,0 +1,102 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Marcelo Raschi
+
+#include "linear_isotropic_damage_plane_strain_2d.h"
+#include "structural_mechanics_application_variables.h"
+
+namespace Kratos
+{
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+LinearIsotropicDamagePlaneStrain2D::LinearIsotropicDamagePlaneStrain2D()
+    : LinearIsotropicDamage3D()
+{
+}
+
+//********************************COPY CONSTRUCTOR************************************
+//************************************************************************************
+LinearIsotropicDamagePlaneStrain2D::LinearIsotropicDamagePlaneStrain2D(
+    const LinearIsotropicDamagePlaneStrain2D &rOther)
+    : LinearIsotropicDamage3D(rOther)
+{
+}
+
+//********************************CLONE***********************************************
+//************************************************************************************
+
+ConstitutiveLaw::Pointer LinearIsotropicDamagePlaneStrain2D::Clone() const
+{
+    return Kratos::make_shared<LinearIsotropicDamagePlaneStrain2D>(LinearIsotropicDamagePlaneStrain2D(*this));
+}
+
+//********************************DESTRUCTOR******************************************
+//************************************************************************************
+
+LinearIsotropicDamagePlaneStrain2D::~LinearIsotropicDamagePlaneStrain2D() = default;
+
+//************************************************************************************
+//************************************************************************************
+
+void LinearIsotropicDamagePlaneStrain2D::CalculateConstitutiveMatrix(
+    Matrix &rConstitTensor,
+    const Properties &rMaterialProperties
+    )
+{

```

Comment:
If this is the linear elastic constitutive matrix you can derive the law from the linear elastic one, in a future PR

Reply:
I'm preparing a followup PR with changes for this and other CLs. I will include this suggestion.